### DISCUSSION TITLE
verify stats

### DISCUSSION THREAD

**Discussion by github-dub** [axboe/fio#1812] [Q&A] [answered]
For the following linux fio command, should I see any verification stats on stdout or json file?  I see the reads, but nothing at all about   verify.

fio --ioengine=libaio --direct=1 --group_reporting=1 \
    --name=fio_file2G \
    --directory=/voltest \
    --size=2G \
    --bs=8k \
    --iodepth=1 \
    --rw=randwrite \
    --rwmixread=100 \
    --rate_iops=100,100 \
    --verify=md5 \
    --verify_only \
    --filename=fio_file2G \
    --aux-path=/home/dub157/
    --output-format=json

**Comment by vincentkfu** [comment, marked as answer]
Fio will print out error messages when it encounters a verify failure. If everything completes successfully then no failures were encountered. You can test this by modifying your file and running the job again.

Verify operations are reads. Your job is a randwrite job with `verify_only`, so fio actually did read and verify your file. You can also run with `--debug=verify,io` to gain more insight into what fio is actually doing.
